src/loaders/trackedEntityLoader.js

```diff
@@ -117,17 +106,22 @@ const trackedEntityLoader = async (config) => {
     }
 
     if (periodType === 'program') {
-        url += `&programStartDate=${startDate}&programEndDate=${endDate}`
+        url += `&enrollmentEnrolledAfter=${startDate}&enrollmentEnrolledBefore=${endDate}`
     } else {
-        url += `&lastUpdatedStartDate=${startDate}&lastUpdatedEndDate=${endDate}`
+        url += `&updatedAfter=${startDate}&updatedBefore=${endDate}`
     }
 
     // https://docs.dhis2.org/master/en/developer/html/webapi_tracker_api.html#webapi_tei_grid_query_request_syntax
     const primaryData = await apiFetch(url)
 
-    const instances = primaryData.trackedEntityInstances.filter(
+    const trackerRootProp =
```

Comment:
I think the comment on line 114 can be deleted

Reply:
Addressed here: https://github.com/dhis2/maps-app/pull/3346/commits/2e17afcaab97cbce1cb836c554aecd31260f8d97